[2024-08-21T06:27:29.173+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-21T06:27:29.290+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_22024-08-21.fetch_the_api scheduled__2024-08-11T00:00:00+00:00 [queued]>
[2024-08-21T06:27:29.351+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_22024-08-21.fetch_the_api scheduled__2024-08-11T00:00:00+00:00 [queued]>
[2024-08-21T06:27:29.358+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-21T06:27:29.978+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): fetch_the_api> on 2024-08-11 00:00:00+00:00
[2024-08-21T06:27:30.194+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4382) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-21T06:27:30.210+0000] {standard_task_runner.py:63} INFO - Started process 4391 to run task
[2024-08-21T06:27:30.215+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_22024-08-21', 'fetch_the_api', 'scheduled__2024-08-11T00:00:00+00:00', '--job-id', '720', '--raw', '--subdir', 'DAGS_FOLDER/final_etl.py', '--cfg-path', '/tmp/tmpo80qx_8a']
[2024-08-21T06:27:30.253+0000] {standard_task_runner.py:91} INFO - Job 720: Subtask fetch_the_api
[2024-08-21T06:27:30.870+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_22024-08-21.fetch_the_api scheduled__2024-08-11T00:00:00+00:00 [running]> on host 7e1333882d71
[2024-08-21T06:27:31.202+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_22024-08-21' AIRFLOW_CTX_TASK_ID='fetch_the_api' AIRFLOW_CTX_EXECUTION_DATE='2024-08-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-11T00:00:00+00:00'
[2024-08-21T06:27:31.211+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-21T06:27:31.242+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-21T06:27:31.245+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/data/config.py']
[2024-08-21T06:27:31.260+0000] {subprocess.py:86} INFO - Output:
[2024-08-21T06:27:31.584+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-21T06:27:31.609+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-21T06:27:31.844+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_22024-08-21, task_id=fetch_the_api, run_id=scheduled__2024-08-11T00:00:00+00:00, execution_date=20240811T000000, start_date=20240821T062729, end_date=20240821T062731
[2024-08-21T06:27:31.933+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-21T06:27:32.123+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-21T06:27:32.135+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
